.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - 'v*.*.*'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        go-version: [ 1.17 ]
```

I propose to use go 1.18
```suggestion
        go-version: [ 1.18 ]
```